src/OrchardCore/OrchardCore.Application.Cms.Targets/ServiceExtensions.cs

```diff
@@ -22,12 +16,9 @@ public static IServiceCollection AddOrchardCms(this IServiceCollection services,
             services.AddAuthentication();
             services.AddModules(modules => 
             {
-                if (configuration != null)
-                {
-                    modules.WithConfiguration(configuration);
```

Comment:
The question is, can any module get the configuration through `IConfiguration` injection without this ... I don't know if we have anything right now in a module that exercises this scenario.

Reply:
> The question is, can any module get the configuration through IConfiguration injection without this ...

Yep. It works in any module Startup constructor or any module component.